DigitalLearningSolutions.Web/Scripts/trackingSystem/reports.ts

```diff
@@ -16,6 +16,9 @@ const activityToggleableRowDisplayTableRow = 'table-row';
 const mobileMaxNumberOfEntriesForActivityGraph = 17;
 const desktopMaxNumberOfEntriesForActivityGraph = 31;
 
+const noActivityId = 'no-activity-message';

```

Comment:
Nitpick - I'd call this `noActivityMessageId`, because it's the id of the no-activity-message.

Reply:
Does it even need to be its own const? just stick the string into the getElementById

Reply:
I think it is useful to have it as a const of its own - if the id of the message ever gets changed it would be good make it as easy as possible to find the value(s) to change in reports.ts (and make it as obvious as possible that it needs to be changed).